projects/js-toolkit/tsconfig.json

```diff
@@ -2,13 +2,13 @@
 	"compileOnSave": true,
 	"compilerOptions": {
 		"esModuleInterop": true,
-		"lib": ["ES2015", "DOM"],
+		"lib": ["ES2015"],
```

Comment:
This might be for a different thread, but any thoughts on upgrading to es2021?

Reply:
Apparently ES2021 is only [fully supported in node v16](https://stackoverflow.com/questions/67371787/what-typescript-configuration-produces-output-closest-to-node-js-16-capabilities), so we may want to defer this, as we usually say that we use node LTS version as reference (which [is now 14](https://nodejs.org/en/)) in all our tooling...